aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -42,7 +42,7 @@ Discover Azure App Service runtime execution limitations enforced by the Azure A
 
 ::: moniker range=">= aspnetcore-2.2"
 
-Runtimes for 64-bit (x64) and 32-bit (x86) apps are present on Azure App Service. The [.NET Core SDK](/dotnet/core/sdk) available on App Service is 32-bit, but you can deploy 64-bit apps using the [Kudu](https://github.com/projectkudu/kudu/wiki) console or via [MSDeploy with a Visual Studio publish profile or CLI command](xref:host-and-deploy/visual-studio-publish-profiles).
+Runtimes for 64-bit (x64) and 32-bit (x86) apps are present on Azure App Service. The [.NET Core SDK](/dotnet/core/sdk) available on App Service is 32-bit, but you can deploy 64-bit apps built locally using the [Kudu](https://github.com/projectkudu/kudu/wiki) console or the publish process in Visual Studio. For more information, see the [Publish and deploy the app](#publish-and-deploy-the-app) section.
```

Comment:
The past few days I was experimenting with a site we're porting to 3.0 preview 7. In so doing, we determined that the `.csproj` file had been set to 64b, and we kept getting ANCM errors on App Service. After a quick check we ascertained that the site was running in 32b. Once we changed it to 64b to match what was in the `.csproj` file it worked. I mention this as it bit us each time we set up a new slot. Somehow, seems we could encourage the customer to look at that setting if they see an ANCM issue. Might be good to show a screen shot in some doc, maybe not here, but somewhere.

Reply:
`<Platform>` property?

The happy path AFAICT is not to set `<Platform>` (and we don't mention it anywhere AFAIK). I had a good outcome only using the switch for **Platform** in  **General settings**. It seems 🤞 to work well that way for an FDD.

It's a candidate for coverage in the troubleshooting topic ... I mean if we **don't** say anything about it here.

Reply:
good call!

Reply:
I'll work it further on https://github.com/aspnet/AspNetCore.Docs/issues/12965.

Anything else on this PR?

Reply:
Nope all good.